api/GrpcHandler.go

```diff
@@ -2,7 +2,7 @@ package api
 
 import (
 	"context"
-	"github.com/devtron-labs/git-sensor/internal/sql"
+	"github.com/devtron-labs/git-sensor/internals/sql"
```

Comment:
`internal` directory - is a special name that do not allow others to import any package under this directory. So after renaming it to `internals` - anyone can import this packages. I am almost sure that this is not a main goal of this PR.